src/mrnet/network/reaction_network.py

```diff
@@ -595,13 +595,13 @@ def get_formula(x):
             return x.formula
 
         def get_Nbonds(x):
```

Comment:
should probably also change this to get_num_bonds

Reply:
Thanks for the catch!